sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/OffsetType.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.EventHubs
+{
+    public enum OffsetType
```

Comment:
Hmm why are the analyzers not complaining about docs missing on public type?

Reply:
good question.